ocaml/xapi/xapi_xenops.ml

```diff
@@ -353,6 +353,7 @@ let builder_of_vm ~__context (vmref, vm) timeoffset pci_passthrough vgpu =
         |false -> true
       end;
       vncterm_ip = None (*None PR-1255*);
+      pci_passthrough = List.mem_assoc "pci" vm.API.vM_other_config;
```

Comment:
I think that the (right) value of `pci_passthrough` is available as an argument to the outer function `builder_of_vm`. So this can just be `pci_passthrough;`.